docs/source/model_definition.md

```diff
@@ -1,26 +1,61 @@
+For more information, refer to the online [appendix](https://github.com/NVIDIA-Merlin/publications/blob/main/2021_acm_recsys_transformers4rec/Appendices/Appendix_A-Techniques_used_in_Transformers4Rec_Meta-Architecture.md) to the preceding paper.
+Categorical input features are optionally normalized (with layer normalization) before aggregation.
+Continuous features should be normalized during feature engineering of the dataset.
 
-Categorical features are represented by embeddings. Numerical features can be represented as a scalar, projected by a fully-connected (FC) layer to multiple dimensions, or represented as a weighted average of embeddings by the technique Soft One-Hot embeddings (more info in our [paper online appendix](https://github.com/NVIDIA-Merlin/publications/blob/main/2021_acm_recsys_transformers4rec/Appendices/Appendix_A-Techniques_used_in_Transformers4Rec_Meta-Architecture.md)). Categorical input features are optionally normalized (with layer normalization) before aggregation.  Continuous features should be normalized during feature engineering of the dataset.
+{class}`TabularSequenceFeatures <transformers4rec.torch.features.sequence.TabularSequenceFeatures>` is the core class of this module.
+This class processes and aggregates all features and outputs a sequence of *interaction embeddings* to be fed into transformer blocks.
+You can create an instance of `TabularSequenceFeatures` automatically from a dataset schema that is generated from NVTabular by using the {func}`from_schema() <transformers4rec.torch.features.sequence.TabularSequenceFeatures.from_schema>` method.
+This method creates the layers that are required to represent the categorical and continuous features in the dataset.
+In addition, you can specify the `aggregation` option of this method to aggregate the sequential features and to prepare masked labels according to the specified sequence masking approach.
 
-The core class of this module is the `TabularSequenceFeatures`, which is responsible for processing and aggregating all features and outputs a sequence of *interaction embeddings* to be fed into transformer blocks. It can be instantiated automatically from a dataset schema (`from_schema()`) generated by [NVTabular](https://github.com/NVIDIA-Merlin/NVTabular), which directly creates all the necessary layers to represent the categorical and continuous features in the dataset. In addition, it has options to aggregate the sequential features, and to prepare masked labels depending on the chosen sequence masking approach (see next section)).
+The following code block shows one way the `TabularSequenceFeatures.from_schema()` method can create the interaction embeddings that are ready for use with Transformer blocks:
 
 ```python
 from transformers4rec.torch import TabularSequenceFeatures
```

Comment:
Several users were asking about how the embedding dimensions of input features are set in`TabularSequenceFeatures`. So perhaps we can expose the default parameter `embedding_dim_default` that sets a fixed dimension for all categorical input features?
```
from transformers4rec.torch import TabularSequenceFeatures
tabular_inputs = TabularSequenceFeatures.from_schema(
        schema,
        embedding_dim_default=128,
        max_sequence_length=20,
        d_output=100,
        aggregation="concat",
        masking="clm")
```